packages/yourloops/components/chat/chat-widget.tsx

```diff
@@ -156,12 +156,15 @@ function ChatWidget(props: ChatWidgetProps): JSX.Element {
   useEffect(() => {
     async function fetchMessages() {
       const messages = await getChatMessages(team.teamId, patient.userid);
+      if (patient.metadata.unreadMessagesSent > 0) {
+        teamHook.markPatientMessagesAsRead(patient);
+      }
```

Comment:
getting messages (through await getChatMessages) marked them as read... So no need for now to make a second api call.